addons/project/tests/test_project_base.py

```diff
@@ -178,6 +178,57 @@ def test_search_favorite_order(self):
         self.assertTrue(self.project_pigs.id < self.project_goats.id)
         self.assertEqual(Project.search(domain, order='id').ids, project_ids)
 
+    @users('bastien')
+    def test_edit_favorite(self):
+        project1 = self.env['project.project'].create({
+            'name': 'Project Test1',
+        })
+        self.assertFalse(project1.is_favorite)
+        project1.is_favorite = True
+        project1.invalidate_recordset(['is_favorite']) # To force 'is_favorite' to recompute
+        self.assertTrue(project1.is_favorite)
+
+        project2 = self.env['project.project'].create({
+            'name': 'Project Test2',
+            'is_favorite': True,
+        })
+        self.assertTrue(project2.is_favorite)
+        project2.is_favorite = False
+        project2.invalidate_recordset(['is_favorite']) # To force 'is_favorite' to recompute
+        self.assertFalse(project2.is_favorite)
+
+    @users('bastien')
+    def test_create_favorite_from_project_form(self):
+        form1 = Form(self.env['project.project'])
```

Comment:
```suggestion
        Project = self.env['project.project']
        form1 = Form(Project)
```
As we use it twice. I know it is a detail, but while we're at it... :)